### PR TITLE
Init Vietnamese localization

### PR DESCRIPTION
**Pull request by truongnh1992**
Init of the Vietnamese translation of the Kubernetes documentation / Website

Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by sftim** [comment]
Good to see another localization proposed!
I don't speak Vietnamese, so
/uncc

**Comment by zacharysarah** [comment]
@truongnh1992 Thanks for opening this PR! As mentioned in https://github.com/kubernetes/org/pull/1314, you need at least two people to begin a new localization.

You'll also need to add a repository label for `language/vi`. You can do that by opening a pull request in the **kubernetes/test-infra** repository, using https://github.com/kubernetes/test-infra/pull/12103 as an example.

Please let me know when you've met these requirements!

/hold

**Comment by truongnh1992** [comment]
@zacharysarah I've already create PR to add a repository label fir `language/vi`
kubernetes/test-infra#14817

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -179,3 +179,7 @@ aliases:
     - femrtnz
     - jcjesus
     - devlware
```
Comment (zacharysarah) [inline OWNERS_ALIASES]: Please add @ngtuna to both of these blocks.

File: config.toml [hunk 2]
```diff
@@ -266,3 +266,10 @@ contentDir = "content/hi"
 [languages.hi.params]
 time_format_blog = "01.02.2006"
 language_alternatives = ["en"]
+
+[languages.vi]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+languageName = "Vietnamese"
+contentDir = "content/vi"
+weight = 3
```
Comment (zacharysarah) [inline config.toml]: weight = 12

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -179,3 +179,9 @@ aliases:
...
+  sig-docs-vi-owners: # Admins for Vietnamese content
+    - truongnh1992
+    - ngtuna
```
Comment (zacharysarah) [inline OWNERS_ALIASES]: Please alphabetize these lists.

**Comment by remyleone** [comment]
https://github.com/kubernetes/org/issues/1315

**Comment by zacharysarah** [comment]
Holding for https://github.com/kubernetes/org/pull/1323

**Comment by truongnh1992** [comment]
> Holding for [kubernetes/org#1323](https://github.com/kubernetes/org/pull/1323)

I've just update the PR kubernetes/org#1323 

/unhold

**Pull request merged**